src/main/java/xyz/nucleoid/plasmid/event/GameEvents.java

```diff
@@ -0,0 +1,72 @@
+package xyz.nucleoid.plasmid.event;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+import net.minecraft.util.Identifier;
+import xyz.nucleoid.plasmid.game.ConfiguredGame;
+import xyz.nucleoid.plasmid.game.GameCloseReason;
+import xyz.nucleoid.plasmid.game.GameSpace;
+import xyz.nucleoid.plasmid.game.StartResult;
+
+/**
+ * Events for games being opened and closed/finishing
+ */
+public final class GameEvents {
+
+    private GameEvents() { }
+
+    public static final Event<GameOpening> OPENING = EventFactory.createArrayBacked(GameOpening.class,
+            listeners -> (gameId, game, oneShot) -> {
+        for (GameOpening listener : listeners) {
+            listener.onGameOpening(gameId, game, oneShot);
+        }
+    });
+
+    public static final Event<GameOpen> OPEN = EventFactory.createArrayBacked(GameOpen.class,
```

Comment:
Might be worth clarifying this as `OPENED`, similar to how server starting/started events are in Fabric API.